### PR TITLE
Add utility function enumerateFiles

### PR DESCRIPTION
**Pull request by maliroteh-sf**

### CODE REVIEW

File: src/common/CommonUtils.ts [hunk 1]
```diff
@@ -221,32 +221,45 @@ export class CommonUtils {
...
-                            msg = `${msg}\nstdout:\n${stdout}`;
-                        }
-
-                        CommonUtils.logger.error(msg);
-                        reject(error);
-                    } else {
-                        resolve({ stdout, stderr });
                     }
-                });
+                );
```
Comment (maliroteh-sf) [inline src/common/CommonUtils.ts:254]: I didn't change any logic here.... it's just formatting/indentation changes

File: src/common/CommonUtils.ts [hunk 2]
```diff
@@ -221,32 +221,45 @@ export class CommonUtils {
...
+
+                if (forwardStdOut) {
+                    prc.stdout?.pipe(process.stdout);
+                }
+
+                if (forwardStdErr) {
+                    prc.stderr?.pipe(process.stderr);
+                }
```
Comment (maliroteh-sf) [inline src/common/CommonUtils.ts:262]: I only added these

File: src/common/__tests__/CommonUtils.test.ts [hunk 3]
```diff
@@ -196,4 +196,110 @@ describe('CommonUtils', () => {
             expect(content).toBe(testContent);
         }
     }, 10000); // increase timeout for this test
+
+    test('enumerateFiles - file as input ', async () => {
```
Comment (sfdctaka) [inline src/common/__tests__/CommonUtils.test.ts:200]: Test! 🎉

**Review by sfdctaka — APPROVED** [review]
LGTM!

**Pull request merged**